### PR TITLE
Turn on NOAA-20 OMPS and PlanetIQ in operations

### PR DESCRIPTION
**Pull request by ADCollard**
Fix file changes to turn on PlanetIQ GPSRO and NOAA-20 OMPS data in operations.

 New feature (non-breaking change which adds functionality)
Tested in operational global workflow.

Checklist

 My code follows the style guidelines of this project
 I have performed a self-review of my own code
 I have commented my code, particularly in hard-to-understand areas
 New and existing tests pass with my changes
 Any dependent changes have been merged and published

### CODE REVIEW

**Comment by ADCollard** [comment]
@RussTreadon-NOAA I am not sure why the scripts change is in this PR.  I thought this was already merged.

File: scripts/exglobal_atmos_analysis.sh
Comment (RussTreadon-NOAA) [inline scripts/exglobal_atmos_analysis.sh]: The atms_n21 and cris-fsr_n21 lines were removed from `exglobal_atmos_analysis.sh` via PR #611.   Why are we seeing them removed again?

A check of [ADCollard:release/gfsda.v16](https://github.com/ADCollard/GSI/tree/release/gfsda.v16) shows 
 the branch is 1 commit behind [NOAA-EMC:release/gfsda.v16](https://github.com/NOAA-EMC/GSI/tree/release/gfsda.v16).  

We should update ADCollard:release/gfsda.v16 with the current head of NOAA-EMC/release/gfsda.v16.  If we do so the change to `exglobal_atmos_analysis.sh` in this PR should disappear.

**Review by RussTreadon-NOAA — APPROVED** [review]
As expected, the only change is the update to the fix submodule hash.   

I cloned `ADCollard:release/gfsda.v16` and `NOAA-EMC:release/gfsda.v16`.   The only differences are in the following files
- `fix/global_convinfo.txt`
- `fix/global_ozinfo.txt`

`ADCollard:release/gfsda.v16` `global_convinfo.txt` places gps 267 in assimilate mode.    The `global_ozinfo.txt` (a) places ompsnp_n20 in assimilate mode and (b) changes omps errors.

These appear to be the correct changes.  Approve.

Merger of this PR into NOAA-EMC:release/gfsda.v16 is awaiting confirmation from @XuanliLi-NOAA  and @HaixiaLiu-NOAA that the fix file changes in this PR are correct.

**Comment by ADCollard** [comment]
@RussTreadon-NOAA Thanks, that seemed to work.

**Comment by RussTreadon-NOAA** [comment]
@ADCollard , I'll create tag `gfsda.v16.3.9` once this PR is merged into `NOAA-EMC:release/gfsda.v16`.   Is `16.3.9` the correct version?

**Comment by ADCollard** [comment]
No, please re-create gfsda.v16.3.8
Thanks!
Andrew.

**Comment by RussTreadon-NOAA** [comment]
Oops, glad I asked.  I thought we had two sequential implementations.  Not so.  We're lumping them together.   That's easier for NCO.

**Comment by ADCollard** [comment]
Yes, I was very surprised when Steven said yes to this!

**Comment by RussTreadon-NOAA** [comment]
Some surprises are good.  I'll merge this PR into `NOAA-EMC:release/gfsda.v16` no later than 3 pm today with or without additional approvals.   I'll create a new `gfsda.v16.3.8` after merging.

**Review by XuanliLi-NOAA — APPROVED** [review]

**Comment by RussTreadon-NOAA** [comment]
Create tag [gfsda.v16.3.8](https://github.com/NOAA-EMC/GSI/releases/tag/gfsda.v16.3.8) as a copy of [release/gfsda.v16](https://github.com/NOAA-EMC/GSI/tree/release/gfsda.v16) at [fe08709](https://github.com/NOAA-EMC/GSI/commit/fe08709d0cf8054acbd5544c17f4d5a7d7aa492d)

**Comment by ADCollard** [comment]
Thankyou @RussTreadon-NOAA !

**Review by HaixiaLiu-NOAA — COMMENTED** [review]
I can now confirm that the global_ozinfo.txt contains correct obs errors for OMPSNP and iuse as well. Approve now.

**Pull request merged**
